themes/src/main/resources-community/theme/base/login/messages/messages_ca.properties

```diff
@@ -19,7 +19,7 @@ registerWithTitleHtml={0}
 loginTitle=Inicia sessi\u00F3 a {0}
 loginTitleHtml={0}
 impersonateTitle={0}\u00A0Personifica Usuari
-impersonateTitleHtml=<strong>{0}</strong> Personifica Usuari</strong>
```

Comment:
strong is already closed earlier

Reply:
Yes. This is an unnecessary tag. Do you think no need to remove?

Reply:
The additional </strong> you've added to the end needs to be removed